playbooks/de-populate-consul.yaml

```diff
@@ -93,3 +93,51 @@
         - {key: "configs/{{environment_name}}/notification-agent/clean/age", value: "{{ notificationagent.clean_age }}"}
         - {key: "configs/{{environment_name}}/notification-agent/clean/enable", value: "{{ notificationagent.clean_enable }}"}
         - {key: "configs/{{environment_name}}/services/port", value: "{{ default_service_port }}"}
+        - {key: "configs/{{environment_name}}/de/default-build-number", value: "{{ de_version_name }}"}
+        - {key: "configs/{{environment_name}}/de/release-version", value: "{{ de_version }}"}
+        - {key: "configs/{{environment_name}}/de/base-url", value: "{{ cas_base }}"}
+        - {key: "configs/{{environment_name}}/de/cas-server-name", value: "{{ de.app_base }}"}
+        - {key: "configs/{{environment_name}}/de/cas-validation", value: "/iplant-cas-ticket-validator"}
+        - {key: "configs/{{environment_name}}/de/cas-logout-url", value: "/iplant-cas-logout"}
+        - {key: "configs/{{environment_name}}/de/cas-app-name", value: "{{ de.description }}"}
+        - {key: "configs/{{environment_name}}/de/cas-login-url", value: "/login"}
+        - {key: "configs/{{environment_name}}/de/authorized-groups", value: "{{ admin_groups }}"}
+        - {key: "configs/{{environment_name}}/de/group-attribute-name", value: "{{ cas_group_attribute }}"}
+        - {key: "configs/{{environment_name}}/de/cas-no-logout-url", value: "{{ cas_no_logout_url }}"}
+        - {key: "configs/{{environment_name}}/de/cas-app-list", value: "{{ cas_app_list }}"}
+        - {key: "configs/{{environment_name}}/de/keepalive-service", value: "{{ de.app_base }}/de/{{ de.rpc_name }}/{{ de.empty_url }}"}
+        - {key: "configs/{{environment_name}}/de/keepalive-target", value: "{{ cas_base }}/login?service={{ de.app_base }}/de/{{ de.rpc_name }}/{{ de.empty_url }}"}
+        - {key: "configs/{{environment_name}}/de/keepalive-interval", value: "90"}
+        - {key: "configs/{{environment_name}}/de/jwt-private-key-path", value: "{{ jwt.signing_key.private }}"}
+        - {key: "configs/{{environment_name}}/de/jwt-private-key-password", value: "{{ jwt.signing_key.password }}"}
+        - {key: "configs/{{environment_name}}/de/twitter-url", value: "https://twitter.com/cyverseorg"}
+        - {key: "configs/{{environment_name}}/de/facebook-url", value: "https://www.facebook.com/CyVerse.org/"}
+        - {key: "configs/{{environment_name}}/de/newsletter-url", value: "http://www.cyverse.org/newsletter"}
+        - {key: "configs/{{environment_name}}/de/ask-url", value: "http://www.cyverse.org/learning-center/ask-cyverse"}
+        - {key: "configs/{{environment_name}}/de/prefix", value: "org.iplantc.services"}
+        - {key: "configs/{{environment_name}}/de/file-identifier", value: "{{ path_list_file_identifier }}"}
+        - {key: "configs/{{environment_name}}/de/email-base", value: "{{ iplant_email.base }}/"}
```

Comment:
This is already above, as `/email/base`, so that should probably be used instead.